lib/Activity/Provider.php

```diff
@@ -75,10 +75,10 @@ public function parse($language, IEvent $event, IEvent $previousEvent = null) {
 				$event->setSubject($l->t('You added an WebAuthn hardware token'));
 				break;
 			case 'webauthn_device_removed':
-				$event->setSubject($l->t('You removed an WebAuthn hardware token'));
```

Comment:
Same here as written in my previous post. 
Please revert and stay with "an".

Reply:
Done 😉.